src/NelsonSiegelSvensson.jl

```diff
@@ -0,0 +1,181 @@
+abstract type ParametricModel end
```

Comment:
I think this wants to be abstract type ParametricModel <: AbtstractYield end` so that various functions in the rest of the package (e.g. `discount`) will work with these new types via the [fallback methods](https://github.com/JuliaActuary/Yields.jl/blob/master/src/generics.jl).

Reply:
Agree.